fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -135,6 +136,7 @@
     private static final String WANT_DIGEST = "Want-Digest";
 
     private static final String DIGEST = "Digest";
+    public static final MediaType DEFAULT_CONTENT_TYPE = RDFMediaType.TURTLE_TYPE;
```

Comment:
While this is important, I think this is only the default content type for rdf sources. For non-rdf sources it should be `application/octet-stream` I would think?

Reply:
will fix.

Reply:
done